spec/models/ext_management_system_spec.rb

```diff
@@ -439,22 +440,17 @@
 
         deliver_queue_message
 
-        expect(MiqQueue.count).to eq(1)
-        expect(MiqQueue.last.deliver_on).to_not be_nil
+        expect(MiqQueue.count).to eq(0)
+        expect(ExtManagementSystem.count).to eq(0)
       end
 
       it "destroys the ems when the active worker shuts down" do
-        refresh_worker = FactoryGirl.create(:miq_ems_refresh_worker, :queue_name => ems.queue_name, :status => "started", :miq_server => server)
+        FactoryGirl.create(:miq_ems_refresh_worker, :queue_name => ems.queue_name, :status => "started", :miq_server => server)
         ems.destroy_queue
 
         deliver_queue_message
 
-        expect(ExtManagementSystem.count).to eq(1)
-
-        refresh_worker.destroy
-
-        deliver_queue_message
-
+        expect(MiqQueue.count).to eq(0)
```

Comment:
I think this test can be merged with the one above since they are effectively testing the same thing now.